cli/src/pcluster/validators/slurm_settings_validator.py

```diff
@@ -99,7 +99,7 @@ def _validate(self, custom_settings: List[Dict], deny_list: List[str], settings_
         if len(denied_settings) > 0:
             settings = ",".join(sorted(denied_settings))
             self._add_failure(
-                f"Using the following custom Slurm settings at {settings_level} level is not allowed: {settings}",
+                f"Using the following custom Slurm settings at {settings_level.value} level is not allowed: {settings}",
```

Comment:
Did you check every Enum we have in the code?

Reply:
No. But now I've checked them all